packages/apps/plugins/plugin-space/.storybook/preview-head.html

```diff
@@ -0,0 +1,17 @@
+<script>
+  window.global = window;
+</script>
+<style>
+  html, body {
+    margin: 0 !important;
+    padding: 0 !important;
+  }
+</style>
+<style>
+  .bg-cubes {
+    background-image: url('cubes.jpg');
+  }
+  .dark .bg-cubes {
+    background-image: url('cubes-dark.jpg');
+  }
```

Comment:
These were a CSS-native pattern with configurable colors: https://projects.verou.me/css3patterns/#japanese-cube

I'd recommend:
- best: use the CSS-native pattern with adjusted colors
- okay: use SVG or PNG for the asset since JPEG introduces compression artifacts